walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
Im not seeing the change associated with this method in this PR. Do you know what might have happened here?

Reply:
@alekseychaikovsky This is a part of the walkthrough, they just wanted us to test it out using arrow functions